ui/app/AppLayouts/Chat/views/ChatView.qml

```diff
@@ -175,21 +175,17 @@ StatusSectionLayout {
     rightPanel: Component {
         id: userListComponent
         UserListPanel {
+            readonly property var usersStore: ChatStores.UsersStore {
```

Comment:
I'm not sure I understand your suggestion. The `usersModule` is for this chat in particular. We cannot use `UsersStore` in AppMain, because it wouldn't have access to `chatContentModule`

Reply:
Got it! The suggestion was not to use the context property in the `ChatView`.